portfolio/src/main/webapp/style.css

```diff
@@ -76,4 +76,4 @@ h3 {
 p, li {
     font-family: 'Lato', sans-serif;
     font-size: 1.5em;
-}
```

Comment:
Please add a newline at the end of files

Reply:
Sorry about that! I didn't realize I had another one

Reply:
Np! usually the editors add it for you. Sorry you have to add it manually